api/routers/edr.py

```diff
@@ -57,7 +57,7 @@
 # We can currently only query data, even if we only need metadata like for this endpoint
 # Maybe it would be better to only query a limited set of data instead of everything (meaning 24 hours)
 async def get_locations(
-    bbox: Annotated[str | None, Query(example="5.0,52.0,6.0,52.1")] = None,
+    bbox: Annotated[str | None, Query(examples="5.0,52.0,6.0,52.1")] = None,
```

Comment:
The `examples` don't seem to be working in the docs, while the `example`. Let's leave this for now, and accept the warnings (or silence them).